http_test.go

```diff
@@ -86,7 +90,34 @@ func TestIteratingByWorker(t *testing.T) {
 	}, testChunks, "should have sorted the metrics by hashes")
 }
 
-func TestServerImport(t *testing.T) {
+func testServerImport(t *testing.T, filename string, contentEncoding string) {
+
+	f, err := os.Open(filename)
+	assert.NoError(t, err, "Error reading response fixture")
+	defer f.Close()
+
+	r := httptest.NewRequest(http.MethodPost, "/import", f)
+	r.Header.Set("Content-Encoding", contentEncoding)
+
+	w := httptest.NewRecorder()
+
+	config := localConfig()
+	s := setupLocalServer(t, config)
+
+	handler := handleImport(&s)
+	handler.ServeHTTP(w, r)
+
+	assert.Equal(t, http.StatusAccepted, w.Code, "Test server returned wrong HTTP response code")
+}
+
+func TestServerImportCompressed(t *testing.T) {
+	// Test that the global veneur instance can handle
+	// requests that provide compressed metrics
+	testServerImport(t, filepath.Join("fixtures", "import.deflate"), "deflate")
```

Comment:
this assumes that the test is invoked from the root directory of the repository. Instead of putting this in a file (and having to deal with finding the file at runtime, which is a pain in compiled languages), I'd prefer to put the string (or byte blob, as the case may be) into a constant in another go file.